_pkgdown.yml

```
@@ -32,6 +32,8 @@ reference:
   - intersect.tbl_lazy
   - mutate.tbl_lazy
   - pull.tbl_sql
+  - rows_update.tbl_lazy
```

There is one help page for all "rows" functions, right?

```suggestion
  - "rows-db"
```